install/running_install.adoc

```diff
@@ -154,20 +154,23 @@ system can run out of files to open and the playbook will fail.
 
 To run the RPM-based installer:
 
-. Run the *_prequisites.yml_* playbook. This must be run only once before
-deploying a new cluster. Use the following command, specifying `-i` if your
-inventory file located somewhere other than *_/etc/ansible/hosts_*:
+. Run the *_prequisites.yml_* playbook. This playbook installs required software
+packages, if any, and modifies the container runtimes. Unless you need to
+configure the container runtimes, run this playbook only one time, before you
+deploy a cluster the first time. Run the following command:
```

Comment:
@kalexand-rh 
Could these two sentences be combined? The _run this playbook_/_run this command_ seems a bit awkward. 
_...run this playbook only one time, before you deploy a cluster the first time using  the following command:_
Or, would that make the sentence too long?

Reply:
I don't think it's too long, but instead of awkwardness from repetition you get awkwardness from rambling.
I might move the restriction to a note under the command.